assets/contact/index.js

```diff
@@ -1 +1,13 @@
 import './less/contact.less';
+
+const checkbox = document.getElementById('id_contact_checkbox');
+const contactName = document.getElementById('id_contact_name');
+const contactMail = document.getElementById('id_contact_email');
+
+checkbox.addEventListener('change', () => {
+  if (checkbox.checked) {
+    contactMail.disabled = contactName.disabled = true;
+  } else {
+    contactMail.disabled = contactName.disabled = false;
```

Comment:
In this case, we don't really need the if-statement at all.

We are already using the `checked` value of the checkbox to determine the `disabled` attribute.
That means we can just directly set it based on the value:
``` javascript
contactMail.disabled = contactName.disabled = checkbox.checked;
```
Or maybe a bit clearer on this form:

``` javascript
contactMail.disabled = checkbox.checked;
contactName.disabled = checkbox.checked;
```